### PR TITLE
Add MSP_SET_FAILSAFE_CONFIG and MSP_FAILSAFE_CONFIG

### PR DESCRIPTION
**Pull request by tricopterY**
Failsafe settings not present in backup
See cleanflight/cleanflight-configurator#143
@hydra - My master branch was so badly out of sync with the cleanflight master, that I had to undo certain changes. Hence, you see all the following mess. I am confident you will ignore the unrelated comments that follow - they are old and I don't know how they got referenced here by github.

### CODE REVIEW

File: src/main/io/serial_msp.c [hunk 1]
```diff
@@ -901,6 +904,14 @@ static bool processOutCommand(uint8_t cmdMSP)
         serialize8(masterConfig.auto_disarm_delay); 
         serialize8(masterConfig.disarm_kill_switch);
         break;
+    case MSP_FAILSAFE_CONFIG:
+        headSerialReply(8);
+        serialize8(masterConfig.failsafeConfig.failsafe_delay);
+        serialize8(masterConfig.failsafeConfig.failsafe_off_delay);
+        serialize16(masterConfig.failsafeConfig.failsafe_throttle);//also in MSP_MISC, just to keep things in sync for now
+        serialize16(masterConfig.rxConfig.rx_min_usec);
+        serialize16(masterConfig.rxConfig.rx_max_usec);
```
Comment (hydra) [inline src/main/io/serial_msp.c]: IMHO rx_min_usec and rx_max_usec should be added to MSP_RX_CONFIG / MSP_SET_RX_CONFIG and not failsafe config.   If you add them to the end you can retain backwards compatibility in MSP_SET_RX_CONFIG by only reading/setting the values depending on the data length.

**Comment by hydra** [comment]
@tricopterY you should be able to squash all your commits into a single commit and update the PR branch to just contain one commit with the changes.

**Comment by hydra** [comment]
@tricopterY I have made the changes, I will push them soon for you.

**Comment by tricopterY** [comment]
@hydra - thanks for the insight and info. It's greatly appreciated.

**Comment by hydra** [comment]
Please check and test the new commands added/updated in 441b58c9c8307db0b5a9ea6f0f668c476d9d67dd

**Comment by tricopterY** [comment]
@hydra - There is a bug:
```C
headSerialReply(8);
//should be :
headSerialReply(12);
```

**Pull request closed without merging**